src/development/troubleshoot.md

```diff
@@ -65,6 +65,17 @@ For a MariaDB database, the command `platform db:size` will give approximate dis
 
 For the most reliable disk usage warnings, we strongly recommend all customers enable [Health notifications](/administration/integrations/notifications.md) on all projects.  That will provide you with a push-notification through your choice of channel when the available disk space on any service drops too low.
 
+## No space left on device
+
+During the build hook, you may run into the following error depending on the size of your application:
+
+```
+W: [Errno 28] No space left on device: ...
+```
+
+The cause of this issue has to do with the amount of disk provided to the build container before it is deployed. Build containers are restricted to 4 GB during build, no matter how much disk has been set aside for the deployed application.
```

Comment:
Subtle wording nitpick here.  It's unclear from the phrasing here (and above) if this is referring to the disk space of the application itself, or the writeable mount.  More precisely, the build env is limited to 4 GB regardless of what the mounted disk size is, and thus by implication the application image is limited to 4 GB, too.  Can we rephrase "for the deployed application" a bit to make that distinction?